src/onAppStartup.cpp

```diff
@@ -58,9 +58,21 @@ static void setRLimits() {
 
     if (getrlimit(RLIMIT_NOFILE, &curr)) throw herr("couldn't call getrlimit: ", strerror(errno));
 
```

Comment:
on the system
```
~# sysctl kern.maxfiles kern.maxfilesperproc kern.openfiles 
kern.maxfiles: 129999
kern.maxfilesperproc: 116991
kern.openfiles: 112
~# ulimit -Hn
116991
~# ulimit -Sn
116991
~# ulimit -n
116991
~# ulimit -Hn
116991
```
refactor for testing to snarf as many handles available up to what was asked for